src/routes/chat/Chat.js

```diff
@@ -55,22 +74,47 @@ function Chat() {
     },
     [dispatch, userInput],
   );
+  const updateUserMessage = useCallback(
+    async event => {
+      event.preventDefault();
+      const oldMessage = await message;
```

Comment:
message is a promise? please rename this variable with something more telling, maybe `messagePromise`.

Reply:
It`s data from redux: const message = useSelector(property('chat.message'));
Before it I used data from server (messageById), when change to data from redux, not to make unnecessary requests to the server.

Reply:
if it's not a promise, you can't await it.